lib/utils.js

```diff
@@ -64,6 +64,7 @@ function seed() {
       'INSERT INTO AcctToProgram (acct_id, program_id) VALUES (7, 1)',
       'INSERT INTO AcctToProgram (acct_id, program_id) VALUES (8, 2)',
       'INSERT INTO AcctToProgram (acct_id, program_id) VALUES (1, 1)',
+      'INSERT INTO AcctToProgram (acct_id, program_id) VALUES (1, 2)',
```

Comment:
I just needed to have some more data for my tests. Should not change anything with your tests.